apps/wallet-dashboard/components/Dialogs/Assets/views/DetailsView.tsx

```diff
@@ -146,14 +146,16 @@ export function DetailsView({ onClose, asset, onSend }: DetailsViewProps) {
                                     {metaKeys.map((aKey, idx) => {
                                         const { value, valueLink } = formatMetaValue(
                                             metaValues[idx],
+                                            true,
```

Comment:
there are still some missing links in this file, such as `nftDisplayData?.projectUrl`